dev/bin/save-loop-data.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# script to generate the loop wallets for testing
+NETWORK=regtest
+mkdir ../lnd/loop/$NETWORK 2>/dev/null
+mkdir ../lnd/loop/server/$NETWORK 2>/dev/null
+
+fetch_loop_data() {
```

Comment:
yeah this script only needs to be ran once and the macaroons are saved. the script was already ran and all the loop macaroons are already saved. this script is kept around just in case we need to regenerate them in the future. i followed the same pattern openoms did with lnd macaroons in the `save-lnd-data.sh` script here https://github.com/GaloyMoney/galoy/blob/main/dev/bin/save-lnd-data.sh.